pkg/meta/utils.go

```diff
@@ -385,14 +386,23 @@ func FastGetSummary(r Meta, ctx Context, inode Ino, summary *Summary, recursive
 	return 0
 }
 
-func fastGetSummary(r Meta, ctx Context, inode Ino, summary *Summary, recursive bool) syscall.Errno {
-	space, _, err := r.GetDirStat(ctx, inode)
+func fastGetSummary(r Meta, ctx Context, inode Ino, attr *Attr, summary *Summary, recursive bool) syscall.Errno {
+	space, inodes, err := r.GetDirStat(ctx, inode)
 	if err != nil {
 		return errno(err)
 	}
 	summary.Size += space
 	summary.Length += space
-	summary.Dirs++
+
+	if !attr.Full {
+		if st := r.GetAttr(ctx, inode, attr); st != 0 {
```

Comment:
Some directory could be removed, we could ignore ENOENT

Reply:
Ignored